packaging/brew/build-bottle.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eux  # Exit on error, print commands
+
+# Create a new tap for the packaging formula
+if brew tap | grep -q zeroc-ice/packaging; then
+    echo "Packaging tap zeroc-ice/packaging already exists"
+else
+    echo "Adding packaging tap zeroc-ice/packaging"
+    brew tap-new --no-git zeroc-ice/packaging
+fi
+
+tap_dir=$(brew --repo zeroc-ice/packaging)
+archive_path=$tap_dir/ice.tar.gz
+
+# Create archive of the current git repository
+cd "$(git rev-parse --show-toplevel)"
+git archive --format=tar.gz -o "$archive_path" HEAD
+archive_hash=$(shasum -a 256 "$archive_path" | cut -d ' ' -f 1)
+
+export ICE_VERSION=${ICE_VERSION:-3.8.0}
```

Comment:
I wonder if is better to fail if not set. We should be explicit about what version to build.

Reply:
Ok, I've changed this for now, but what's is there any difference in this vs what we're doing for dpkg by reading from a file is unset.

Reply:
You are right, not really different.

The caveat with debian is that we need to update the changelog. For the release version we don't want the changelog entry to be an automatic update.